[2025-10-06T02:27:34.362+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-10-06T02:27:34.367+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: StockPrice_ETL_v3.load_to_snowflake manual__2025-10-06T02:27:32.381124+00:00 [queued]>
[2025-10-06T02:27:34.369+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: StockPrice_ETL_v3.load_to_snowflake manual__2025-10-06T02:27:32.381124+00:00 [queued]>
[2025-10-06T02:27:34.369+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-10-06T02:27:34.372+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): load_to_snowflake> on 2025-10-06 02:27:32.381124+00:00
[2025-10-06T02:27:34.375+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=28310) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-10-06T02:27:34.376+0000] {standard_task_runner.py:72} INFO - Started process 28313 to run task
[2025-10-06T02:27:34.376+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'StockPrice_ETL_v3', 'load_to_snowflake', 'manual__2025-10-06T02:27:32.381124+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/create_task.py', '--cfg-path', '/tmp/tmpm8z1oqpc']
[2025-10-06T02:27:34.377+0000] {standard_task_runner.py:105} INFO - Job 25: Subtask load_to_snowflake
[2025-10-06T02:27:34.392+0000] {task_command.py:467} INFO - Running <TaskInstance: StockPrice_ETL_v3.load_to_snowflake manual__2025-10-06T02:27:32.381124+00:00 [running]> on host 3e2b30b430ab
[2025-10-06T02:27:34.421+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='StockPrice_ETL_v3' AIRFLOW_CTX_TASK_ID='load_to_snowflake' AIRFLOW_CTX_EXECUTION_DATE='2025-10-06T02:27:32.381124+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-06T02:27:32.381124+00:00'
[2025-10-06T02:27:34.421+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-10-06T02:27:34.425+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-10-06T02:27:34.425+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.10.14-linuxkit-aarch64-with-glibc2.36
[2025-10-06T02:27:34.426+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-10-06T02:27:34.426+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-10-06T02:27:35.136+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-10-06T02:27:35.344+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-10-06T02:28:56.723+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-10-06T02:28:56.725+0000] {logging_mixin.py:190} INFO -  Successfully inserted 100 records into Raw.stock_table
[2025-10-06T02:28:56.726+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-10-06T02:28:56.736+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-10-06T02:28:56.737+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=StockPrice_ETL_v3, task_id=load_to_snowflake, run_id=manual__2025-10-06T02:27:32.381124+00:00, execution_date=20251006T022732, start_date=20251006T022734, end_date=20251006T022856
[2025-10-06T02:28:56.751+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-10-06T02:28:56.759+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-06T02:28:56.760+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
